### PR TITLE
Datasources defined in Application.cfc always get a timezone set LDEV-2902

### PR DESCRIPTION
**Pull request by cfmitrah**
https://luceeserver.atlassian.net/browse/LDEV-2902

### CODE REVIEW

File: test/tickets/LDEV2902/Application.cfc [hunk 1]
```diff
@@ -0,0 +1,69 @@
+component {
+
+	msSQL = getCredencials();
```
Comment (jbampton) [inline test/tickets/LDEV2902/Application.cfc:3]:
```suggestion
	msSQL = getCredentials();
```

File: test/tickets/LDEV2902/Application.cfc [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+		 class: 'com.microsoft.sqlserver.jdbc.SQLServerDriver'
+		, bundleName: 'mssqljdbc4'
+		, bundleVersion: '4.0.2206.100'
+		, connectionString: 'jdbc:sqlserver://'&#msSQL.server#&':'&#msSQL.port#&';DATABASENAME='&#msSQL.database#&';sendStringParametersAsUnicode=true;SelectMethod=direct'
+		, username: #msSQL.username#
+		, password: #msSQL.password#
+		
+	};
+
+	private struct function getCredencials() {
```
Comment (jbampton) [inline test/tickets/LDEV2902/Application.cfc:38]:
```suggestion
	private struct function getCredentials() {
```

File: test/tickets/LDEV2902/Application.cfc [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+		// getting the credetials from the enviroment variables
```
Comment (jbampton) [inline test/tickets/LDEV2902/Application.cfc:39]:
```suggestion
		// getting the credentials from the environment variables
```

File: test/tickets/LDEV2902/Application.cfc [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+			!isNull(server.system.environment.MSSQL_PASSWORD) && 
+			!isNull(server.system.environment.MSSQL_PORT) && 
+			!isNull(server.system.environment.MSSQL_DATABASE)) {
+			msSQL.server=server.system.environment.MSSQL_SERVER;
+			msSQL.username=server.system.environment.MSSQL_USERNAME;
+			msSQL.password=server.system.environment.MSSQL_PASSWORD;
+			msSQL.port=server.system.environment.MSSQL_PORT;
+			msSQL.database=server.system.environment.MSSQL_DATABASE;
+		}
+		// getting the credetials from the system variables
```
Comment (jbampton) [inline test/tickets/LDEV2902/Application.cfc:53]:
```suggestion
		// getting the credentials from the system variables
```

File: test/tickets/LDEV2902/test.cfm [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+	if (form.scene == 3){
+		try{
+			writeOutput(getApplicationSettings().datasources.testemptyTimeZone.timezone);
+		}
+		catch(any e){
+			writeOutput(e.message);
+		}
+	}
+</cfscript>
+
```
Comment (jbampton) [inline test/tickets/LDEV2902/test.cfm:25]:
```suggestion
```

**Review by jbampton — CHANGES_REQUESTED** [review]

**Comment by michaeloffner** [comment]
manually adapted the PR

**Pull request closed without merging**